src/test/lib/operators/delete_test.cpp

```diff
@@ -296,9 +296,9 @@ TEST_F(OperatorsDeleteTest, UseTransactionContextAfterCommit) {
   t1_context->commit();
 
   auto delete_op2 = std::make_shared<Delete>(validate1);
-  delete_op->set_transaction_context(t1_context);
+  delete_op2->set_transaction_context(t1_context);
```

Comment:
Oupsi. I wish we had enabled clang-tidy for the entire code base from the beginning.